### PR TITLE
Move the “…” menu to the budget name

### PR DESCRIPTION
**Pull request by j-f1**
Rationale:

- this further differentiates the “…” menu in the account view from the budget file menu
- this matches lots of other UIs where top-level actions go under the name of the thing (including Discord)
- renaming a budget isn’t a very common action<sup>[*citation needed*]</sup>, so making it require an extra click isn’t too bad (and arguably it’s more discoverable now that you can get to it by clicking on the visible down arrow indicator and then choosing the “rename” option

Screenshot_2023-01-13 17 38 45

### CODE REVIEW

**Comment by rich-howell** [comment]
Settings appears broken? Clicking it does nothing at the point of review. 

Also `<<` is back, is this because an upstream PR to replace that is yet to be merged?

**Comment by j-f1** [comment]
> Settings appears broken? Clicking it does nothing at the point of review.

fixed!

> Also `<<` is back, is this because an upstream PR to replace that is yet to be merged?

I haven’t opened a PR to change that yet.

**Comment by TechwizEE** [comment]
I've really enjoyed the "hidden" renaming feature for the budget name.  I'd prefer more of the "flow" where a default action (edit title) can be hinted at by hovering and then activated by clicking.  Burying this into menus starts to make things feel "nested" and removes some of the "magic" for me. 

My 2cents... I'd prefer the "..." menu be left as it is.

**Comment by rich-howell** [comment]
> I've really enjoyed the "hidden" renaming feature for the budget name. I'd prefer more of the "flow" where a default action (edit title) can be hinted at by hovering and then activated by clicking. Burying this into menus starts to make things feel "nested" and removes some of the "magic" for me.
> 
> My 2cents... I'd prefer the "..." menu be left as it is.

Having a menu of options feels like it is more natural for non tech folk - as I have said before my mum (as an example) wouldn't know what `...` even means or that is where you go to get to a menu and that has to be what we think about when it comes to these decisions is the design of the UX encompassing of all? If not it needs addressing, if it is and everyone knows that `...` is a menu of items then leave it as it is.

**Comment by j-f1** [comment]
I also liked the renaming feature, but I feel like renaming the budget is a really rare operation and I’m hopeful that this makes the other actions more visible and invites the user to feel safe to click in a way that the previous UI of a button with no apparent label doesn’t.

File: packages/desktop-client/src/components/SidebarWithData.js [hunk 1]
```diff
@@ -1,23 +1,58 @@
...
+        dispatch(closeBudget());
+        break;
+      default:
+    }
+  }
+
+  let items = [
+    { name: 'rename', text: 'Rename Budget' },
+    { name: 'settings', text: 'Settings' },
+    { name: 'help', text: 'Help' },
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/SidebarWithData.js:53]: **🔨 warning:** ‏the `help` link should either be hidden when using the electron app, or it should be made to work (currently it won't work).

To check if the app is electron you can use `!Platform.isBrowser`

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**